### PR TITLE
[WR20] Add accessor by name

### PR DESCRIPTION
**Pull request by perrettecl**
Add accessor to get a document by name

Rework configuration to be defined by portfolio instead of global

Signed-off-by: Clement Perrette <clementperrette@eaton.com>

### CODE REVIEW

File: src/fty_security_wallet_server.cc [hunk 1]
```diff
@@ -75,28 +78,44 @@ std::string SecurityWalletServer::handleGetListPortfolio(const Sender & /*sender
...
     /*
-     * No parameters for this command.
+     * Parameters for this command:
      * 
+     * 0. name of the portfolio
      */
+
+    if(params.size() != 1)
+    {
+        throw SecwBadCommandArgumentException("Command need at 1 argument");
```
Comment (jana-rapava) [inline src/fty_security_wallet_server.cc]: "Command needs at least 1 argument"

File: src/fty_security_wallet_server.cc [hunk 2]
```diff
@@ -167,31 +186,97 @@ std::string SecurityWalletServer::handleGetDocumentWithoutSecret(const Sender &
...
+    /*
+     * Parameters for this command:
+     * 
+     * 0. name of the portfolio
+     * 1. document name
+     */
+
+    if(params.size() != 2)
+    {
+        throw SecwBadCommandArgumentException("Command need at 2 arguments");
```
Comment (jana-rapava) [inline src/fty_security_wallet_server.cc]: "Command needs at least 2 arguments"

**Review by jana-rapava — APPROVED** [review]

File: src/selftest-ro/configuration.json [hunk 3]
```diff
@@ -1,32 +1,54 @@
...
+				"supported_types": [
+					"Snmpv3",
+					"Snmpv1",
+					"UserAndPassword"
+				]
+			},
+			{
+				"usage_id": "mass_device_management",
+				"supported_types": [
+					"Snmpv3",
```
Comment (FrancoisRegisDegott-eaton) [inline src/selftest-ro/configuration.json:16]: "mass device management" could be changed to "mass-management" for WR20.
For now, the mass management service never connect to device snmp servers.

**Review by FrancoisRegisDegott-eaton — APPROVED** [review]

File: src/selftest-ro/configuration.json (hunk 3 above)
Comment (perrettecl) [reply src/selftest-ro/configuration.json:16]: That's a file for unit test. No need to change. The run-time configuration file is going to be moved to BIOS/bios-image or some other image preparation scripts

**Pull request merged**